### PR TITLE
Addes Api controllers and genarate the JWT token

### PR DESCRIPTION
**Pull request by andersonlebon**
1. 

- [ ]  Generate User controllers
- [ ]  Generate House controllers
- [ ]  Generate Reservation controllers

2. 

- [ ]  Check the user route
- [ ]  Create the user endpoints

### CODE REVIEW

File: app/controllers/reservations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    render json: @reservations
+  end
+
+  def show
+    @reservation = Reservation.where(id: params[:id]).first
+    render json: @reservation
+  end
+
+  def create
+    @user = User.where(params[:user_id])
```
Comment (toliboff) [inline app/controllers/reservations_controller.rb]: Please, pass column name in where and select the first user from query result

**Review by toliboff — CHANGES_REQUESTED** [review]
Hi @andersonlebon 
REQUIRED CHANGES 
Please fix it before merging

**Review by toliboff — APPROVED** [review]
Hi @andersonlebon 
Good job! It is time to merge it

**Comment by andersonlebon** [comment]
##  Thanks

**Pull request merged**